app/components/widgets/forms/link-input.js

```diff
@@ -56,10 +56,17 @@ export default class LinkInput extends Component {
   linkNameObserver() {
     const link = this.linkName;
     if (socialPlatforms.includes(link)) {
-      this.set('segmentedLink', {
-        protocol : `https://${link}.com/`,
-        address  : ''
-      });
+      if (link === 'weibo') {
+        this.set('segmentedLink', {
+          protocol : `https://${link}.com/u/`,
+          address  : ''
+        });
```

Comment:
This won't do. It is already highly complex, this should go as a prefix

Reply:
should just left as https://${link}.com/ for all

Reply:
No, this should not happen here at all